src/app/pages/home/services/tasks.service.ts

```diff
@@ -127,6 +128,10 @@ export class TasksService {
     return task.completed && this.isToday(task.timeCompleted)
   }
 
+  wasTaskValidToday(task) {
+    return this.isToday(task.timestamp)
```

Comment:
what is the reason for this check?

Reply:
will this still work if completion window is long, so the task timestamp is before today?